src/test/java/com/google/jenkins/plugins/k8sengine/KubernetesEngineBuilderPipelineIT.java

```diff
@@ -179,6 +206,7 @@ private static void kubectlDelete(
             .workspace(workspace)
             .launcher(launcher)
             .kubeConfig(kubeConfig)
+            .namespace("")
```

Comment:
This should probably be a function param so none default namespaces get cleaned up as well.